cellprofiler/modules/measureimageoverlap.py

```diff
@@ -45,12 +45,15 @@
 in the ground truth will be considered a “false negative” (since it was
 labeled as part of the background, but should not be).
 
+For 3D images, the image is turned into a long 2D image and the overlap is
```

Comment:
Unfortunately, it isn't clear how/why it's turned into a long 2D image (not sure what that is). Also grammar of is/be is messed up somehow.